### DISCUSSION TITLE
Strange step in my velocity profiles

### DISCUSSION THREAD

**Discussion by justhawk98** [Exawind/amr-wind#1118] [Q&A] [answered]
Hi everyone,
Recently, I've been trying to recreate results given in [this paper](https://onlinelibrary.wiley.com/doi/abs/10.1002/we.1747) by @tonyinme since he uses LES and an actuator line model to represent the NREL 5MW, something I am also trying to do. I've gone through several iterations and am getting much better results than when I started but I am still seeing a strange step in my velocity profiles at around 0.25 rotor diameters. See below:

![xd1](https://github.com/Exawind/amr-wind/assets/113199305/0e00ad44-9df0-46fc-970e-1ff4b465a0dd)
![xd3](https://github.com/Exawind/amr-wind/assets/113199305/4beda307-b4fc-4677-b464-5c3372182d05)

Has anyone else encountered this or know what could be causing it? Could it just be a function of my OpenFAST files? I've attached a .zip file containing my AMR-Wind input as well as my OpenFAST files. I use the freestream model so anyone should be able to run it without doing a precursor. Also shown below are a snapshot of the x-velocity after 720 seconds of runtime and a screenshot of my mesh. Any advice is appreciated.

![velx](https://github.com/Exawind/amr-wind/assets/113199305/f042b950-42d2-4dbc-9586-1f21a069f9a4)
![MESH](https://github.com/Exawind/amr-wind/assets/113199305/52f44037-ede8-403c-b158-78d87db5a0d3)

[step_report.zip](https://github.com/user-attachments/files/16114871/step_report.zip)

**Comment by marchdf** [comment]
@tonyinme any thoughts on the above?

**Comment by tonyinme** [comment, marked as answer]
Hi @justhawk98 , your input files look consistent with the simulation setup. 

Your input files have epsilon = 4.2 m. So you should be comparing your data to the top plot in Figure 8. 

Your velocity profiles (orange) look consistent with Figure 8 of that paper. 
The blue line looks different. Where is the data for the blue line coming from?

Also, I think you should set the tower points in amr-wind to zero, or to the same number in the aerodyn input files.
`Actuator.T0.num_points_tower             = 10`

Let me know if that helps.
